lib/authenticated-api/authenticated-api.ts

```diff
@@ -104,6 +105,12 @@ export class AuthenticatedApi extends cdk.Construct {
         }
       );
 
+      if (routeProps.lambdaProps.policyStatements) {
+        for (const statement of routeProps.lambdaProps.policyStatements) {
+          routeLambda.role?.addToPolicy(statement);
+        }
+      }
```

Comment:
Pacing through a `role` didn't work well. It meant the client app had to setup everything automatically set on the role by the CDK. This was problematic.

Instead, pass in policy statements to be added to the role created by the CDK.